packages/common/core-utils/src/test/tsconfig.json

```diff
@@ -4,6 +4,7 @@
 		"rootDir": "./",
 		"outDir": "../../lib/test",
 		"types": ["mocha", "node"],
+		"noUnusedLocals": false,
```

Comment:
Ideally let's fix these up rather than disabling the rule

Reply:
The events test file includes classes to verify compilation of examples from the EventEmitter documentation comments. These classes aren’t used in any tests, so `noUnusedLocals` was disabled. Should I remove them?

https://github.com/microsoft/FluidFramework/pull/23037/files#diff-71d9a863321281adbe406c4c8a02f19a88727a399f809143c396219a59eb54f7

Reply:
One option would be to just `export` them from that file (but not export them any further).

Not sure if others have different preferences, but that's what I would do at least.

Reply:
Exporting would be alright, or just make dummy usages of them below with an explanatory comment or something.